### PR TITLE
New package: apache-jmeter-5.1.1

### PR DESCRIPTION
**Pull request by knusbaum**

### CODE REVIEW

File: srcpkgs/apache-jmeter/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+do_build() {
+	ant install
+}
+
+do_install() {
+	rm -Rf src test build
+
+	vlicense LICENSE
+	vmkdir /opt/apache-jmeter
```
Comment (the-maldridge) [inline srcpkgs/apache-jmeter/template]: /opt is disallowed for package owned files, please pick a new directory.  Generally /usr/share/<pkgname> is a good choice.

File: srcpkgs/apache-jmeter/template [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	vcopy . /opt/apache-jmeter
+
+	vmkdir /usr/bin
+	ln -s /opt/apache-jmeter/bin/jmeter ${DESTDIR}/usr/bin/jmeter
+}
+
+apache-jmeter-doc_package() {
```
Comment (the-maldridge) [inline srcpkgs/apache-jmeter/template]: Is there sufficient documentation to justify the split?  Normally we'd keep the docs with the package unless they're huge.

File: srcpkgs/apache-jmeter/template (hunk 1 above)
Comment (Duncaen) [reply srcpkgs/apache-jmeter/template]: Not for "binary" files, `/usr/lib` or maybe even `/usr/libexec`.

File: srcpkgs/apache-jmeter/template (hunk 2 above)
Comment (knusbaum) [reply srcpkgs/apache-jmeter/template]: They're 57MB unpacked. It seemed sizeable to me, but I'm happy to leave them in the main package if you want. Please let me know.

File: srcpkgs/apache-jmeter/template (hunk 1 above)
Comment (the-maldridge) [reply srcpkgs/apache-jmeter/template]: oh right, I forgot about that that the "compiled" artifacts will wind up in here.

File: srcpkgs/apache-jmeter/template (hunk 2 above)
Comment (the-maldridge) [reply srcpkgs/apache-jmeter/template]: Hmm, that is sizeable, go ahead and leave it split.

File: srcpkgs/apache-jmeter/template (hunk 1 above)
Comment (knusbaum) [reply srcpkgs/apache-jmeter/template]: I'll put them in `/usr/libexec`.

**Comment by knusbaum** [comment]
I made the changes. Let me know if more are needed.

**Pull request merged**